webpack/assets/javascripts/react_app/components/ForemanModal/ForemanModalHooks.js

```diff
@@ -0,0 +1,8 @@
+import { useContext } from 'react';
+import { ModalContext } from './ForemanModal';
+
+// Because enzyme doesn't support useContext yet
+export const useModalContext = () => useContext(ModalContext);
```

Comment:
Since it is only for jest, would it make sense putting it inside the test file?

Reply:
@sharvit Perhaps, but then we would have `ForemanModalFooter.js` and `ForemanModalHeader.js` importing `useModalContext` from the test file, which would feel weird.

Reply:
Also I think leaving it here will help establish the file naming and structure convention for custom Hooks :)

Reply:
Got it, make sense :+1: 

Notice you have an unhealthy loop in your imports.
**`ForemanModal`** -> `ForemanModalHeader` -> `ForemanModalHooks` -> **`ForemanModal`**

In this case, would it make sense creating a dedicated file for the `ModalContext`?